recipes-app/iot2050-firmware-update/files/iot2050-firmware-update

```diff
@@ -601,14 +693,12 @@ def main(argv):
         ====================================
 Hit any Key to Exit:
         '''
```

Comment:
Put this just before the input()

Reply:
Or after upgrading failed, hint the user the rollback is about to be performed, then after rollback, ask user to hit any key.